releasenotes/releasenotes_v5.rst

```diff
@@ -0,0 +1,165 @@
+.. _vonk_releasenotes_history_v5:
+
+Current Firely Server release notes (v5.x)
+==========================================
+
+.. _vonk_releasenotes_5_0_0:
+
+Release 5.0.0-beta1, TBD, 2023
+------------------------------
+.. attention::
+    This is a beta release of Firely Server 5.0.0. Although the core functionality remains fully intact, parts of the public API have been removed or heavily modified. Please consult the list under section 'Plugin and Facade' and check whether your implementation is affected by these changes.
+
+Feature
+^^^^^^^
+
```

Comment:
https://firely.atlassian.net/browse/VONK-3923 seems to be missing

Reply:
I cannot find a mention of https://firely.atlassian.net/browse/VONK-3924

Reply:
Added both!